cmd/codegen/arguments_wrapper.go

```diff
@@ -24,8 +24,6 @@ type argumentWrapper func(arg ArgDef) ArgumentWrapperData
 
 func argWrapper(argType string) (wrapper argumentWrapper, err error) {
 	argWrapperMap := map[string]argumentWrapper{
-		"char":                     simpleW("rune", "C.char"),
-		"char[5]":                  simplePtrArrayW(5, "C.char", "rune"),
```

Comment:
according to my [google search results](https://www.google.com/search?q=what+is+bit+size+of+char+type+in+C) it seems that ith should be OK - the table says that char size is 32 so it should be the same as go's rune

![image](https://user-images.githubusercontent.com/73652197/235301706-9746fa06-e5f3-4aaf-b42a-3e4372fcf22e.png)